pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/elastic/migration/BaseMigrator.java

```diff
@@ -42,15 +43,19 @@ protected void fillBulkRequestWithTransformedMapAndExport(Map<String, Object> pa
         while (participantsIterator.hasNext()) {
             Map.Entry<String, Object> entry = participantsIterator.next();
             String participantId = entry.getKey();
-            if (ParticipantUtil.isGuid(participantId)) {
-                Object participantDetails = entry.getValue();
-                transformObject(participantDetails);
-                Map<String, Object> finalMapToUpsert = generate();
-                bulkExportFacade.addDataToRequest(finalMapToUpsert, participantId);
-            }
-            if (isReadyToExport(participantsIterator)) {
-                totalExported += bulkExportFacade.executeBulkUpsert();
-                bulkExportFacade.clear();
+            try {
+                if (ParticipantUtil.isGuid(participantId)) {
+                    Object participantDetails = entry.getValue();
+                    transformObject(participantDetails);
+                    Map<String, Object> finalMapToUpsert = generate();
+                    bulkExportFacade.addDataToRequest(finalMapToUpsert, participantId);
+                }
+                if (isReadyToExport(participantsIterator)) {
+                    totalExported += bulkExportFacade.executeBulkUpsert();
+                    bulkExportFacade.clear();
+                }
+            }catch (Exception e){
+                throw new DsmInternalError(String.format("Error occurred while trying to export data for participant %s", participantId), e);
```

Comment:
Could we log more than just the participant id?  Is `participantDetails` too much?

Reply:
yes of course

Reply:
just a note - that can have phi so we probably should remove that before it makes it to production